yamale/tests/test_meta_test.py

```diff
@@ -75,3 +75,21 @@ def runTest(self):
         validators = DefaultValidators.copy()
         validators['card'] = Card
         self.assertTrue(self.validate(validators))
+
+
+class TestBadRequiredYaml(YamaleTestCase):
+    base_dir = data_folder
+    schema = 'meta_test_fixtures/schema_required_bad.yaml'
+    yaml = 'meta_test_fixtures/data_required_bad.yaml'
+
+    def runTest(self):
+        self.assertRaises(ValueError, self.validate)
+
+
+class TestGoodRequiredYaml(YamaleTestCase):
+    base_dir = data_folder
+    schema = 'meta_test_fixtures/schema_required_good.yaml'
+    yaml = 'meta_test_fixtures/data_required_good.yaml'
+
+    def runTest(self):
+        self.assertTrue(self.validate())
```

Comment:
Very minor, but it looks like you don't have a new line at the end of this file.  You should turn on auto new line for your editor.

Reply:
Thank you for the tip! Found the setting in VSCode and turned it on.